test.sql

```diff
@@ -1,11 +1,21 @@
-\c nc_games
-
-SELECT reviews.*, COUNT(comments.review_id) AS COMMENT_COUNT
-FROM reviews
-LEFT JOIN comments
-ON comments.review_id = reviews.review_id
-GROUP BY reviews.review_id
-ORDER BY reviews.created_at DESC
+\c nc_games_test
+
+INSERT INTO comments
+    (author, body, review_id)
+VALUES
+    ('mallionaire', 'This is a test review', 1)
+RETURNING *;
+
+SELECT * FROM comments
+
```

Comment:
try not to push these files to your github!

only `got add` the right files and/or add these files to your `.gitignore` just in case!